azerothcore/azerothcore-wotlk
# fix(DB/Gameobjects): Added missing posts to Alterac Valley banners in…

… cities.

Fixes #7320



## Issues Addressed:

- Closes #7320
- Closes https://github.com/chromiecraft/chromiecraft/issues/1397

## Tests Performed:

- Tested ingame in Stormwind

## How to Test the Changes:

If Call To Arms: Alterac Valley is not active:
1. `.event stop xx` where xx is the id of active call to arms event You can find it by `.event active`
2. `.event start 18`


1. `.go object 151528`
2. See that they do have the wooden part on which they are placed


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
